src/main/java/org/schemaspy/DbDriverLoader.java

```diff
@@ -105,14 +108,20 @@ public Connection getConnection(Config config, String connectionURL,
      * @throws MalformedURLException
      */
     protected Driver getDriver(String driverClass, String driverPath) throws MalformedURLException {
-        List<URL> classpath = getExistingUrls(driverPath);
+        Set<URL> classpath = getExistingUrls(driverPath);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Use the URI class instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2112)